modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -585,6 +592,15 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
     return $ss->fetch($file);
 }
 
+/**
+ *  Convert a bracketed variable into a string that can become a JS variable
+ *
+ * @param $variable
```

Comment:
What is the type of $variable? Please put the type in the PHP doc. It helps our IDE's pick up on potential problems.